### PR TITLE
TrilinosCouplings: Fixing bug in STK driver

### PR DESCRIPTION
**Pull request by csiefer2**
Originally, this code assembled the same element over and over again.  Oops

### CODE REVIEW

File: packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp [hunk 1]
```diff
@@ -167,6 +169,12 @@
 #include "stk_mesh/base/FieldBase.hpp"  // for field_data, etc
 #include "stk_mesh/base/Types.hpp"      // for BucketVector, EntityId
 
+#ifdef HAVE_TRILINOSCOUPLINGS_MUELU
+#include "TrilinosCouplings_Statistics.hpp"
+#else 
+#error "BORK"
```
Comment (GrahamBenHarper) [inline packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp:175]: :tada:

**Review by GrahamBenHarper — APPROVED** [review]
Lookin good

File: packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp (hunk 1 above)
Comment (csiefer2) [reply packages/trilinoscouplings/examples/scaling/example_Poisson_stk.cpp]: Oops!  Fixed.

**Review by GrahamBenHarper — APPROVED** [review]
:wink:

**Review by GrahamBenHarper — APPROVED** [review]

**Review by GrahamBenHarper — DISMISSED** [review]

**Review by GrahamBenHarper — APPROVED** [review]

**Pull request merged**